amxmodx/newmenus.cpp

```diff
@@ -1076,6 +1076,99 @@ static cell AMX_NATIVE_CALL menu_setprop(AMX *amx, cell *params)
 	return 1;
 }
 
+static cell AMX_NATIVE_CALL menu_getprop(AMX *amx, cell *params)
+{
+	GETMENU(params[1]);
+
+	int len = params[0] / sizeof(cell);
```

Comment:
`len` is not really a good variable name.  It could be `numParams` or `count` or something like that.

Reply:
@Arkshine Done.